cmd/loop/loopin.go

```diff
@@ -18,6 +18,14 @@ var (
 		Usage: "the pubkey of the last hop to use for this swap",
 	}
 
+	confTargetFlag = cli.Uint64Flag{
+		Name: "conf_target",
+		Usage: "the target number of blocks the on-chain " +
+			"htlc broadcast by the swap client should " +
+			"confirm within",
+		Value: uint64(loop.DefaultHtlcConfTarget),
```

Comment:
nit: same as my previous comment, maybe it's best to avoid these unnecessary casts by simply using `cli.IntFlag` instead.